agoradatatools/etl/transform.py

```diff
@@ -361,11 +375,15 @@ def apply_custom_transformations(datasets: dict, dataset_name: str, dataset_obj:
         return transform_rna_seq_data(datasets=datasets)
     elif dataset_name == 'gene_info':
         return transform_gene_info(datasets=datasets,
-                                   adjusted_p_value_threshold=dataset_obj['custom_transformations']['adjusted_p_value_threshold'],
-                                   protein_level_threshold=dataset_obj['custom_transformations']['protein_level_threshold'])
+                                   adjusted_p_value_threshold=dataset_obj['custom_transformations'][
+                                       'adjusted_p_value_threshold'],
+                                   protein_level_threshold=dataset_obj['custom_transformations'][
+                                       'protein_level_threshold'])
```

Comment:
Same here, can you put these back into one line each?

Reply:
done!